tools/build_script_generator/common.py

```diff
@@ -260,7 +260,8 @@ def common_compiler_flags(compiler, target):
         # "-Wnon-virtual-dtor",
         # "-Wold-style-cast",
         "-fstrict-enums",
-        "-std=c++17",
+        "-std=c++20",
+        "-Wno-volatile",  # volatile is deprecated in C++20 but lots of our external code uses it...
```

Comment:
Note that not every thing volatile is deprecated, but nearly every peace of exiting embedded vendor code.

Reply:
Yes, in theory a good idea, just not very practical: https://github.com/modm-io/modm/pull/521#issuecomment-748393084